IOS/SwiftSeedProject/SwiftSeedProject/Domain/Models/Location.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Location.swift
+//  SwiftSeedProject
+//
+//  Created by Brian Sztamfater on 5/4/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+struct Location {
+    
+    var lat: String
+    var lng: String
```

Comment:
And why are those strings? That's susceptible to localization errors when decimal separator is changed from comma to dot or vice versa. And it's hard to deal with comparisons/precision/thresholds (e.g. know if a certain location is inside a certain rectangle, it's a common scenario)